explorer/interpreter/resolve_names.cpp

```diff
@@ -148,6 +153,11 @@ auto NameResolver::ResolveQualifier(DeclaredName name,
       return ProgramError(name.source_loc())
              << PrintAsID(node.base()) << " cannot be used as a name qualifier";
     }
+
+    if (trace_stream_->is_enabled()) {
+      *trace_stream_ << "--- resolved qualifier `" << qualifier << "` for "
+                     << "\n";
```

Comment:
This message seems to be missing the name of the entity whose qualifier we resolved. Though perhaps we don't need this trace message at all with the tweak to `Resolve`'s output I suggested above.

Reply:
Yes, I'll remove this in next commit.